apps/payments/next/config.ts

```diff
@@ -0,0 +1,478 @@
+      doc: 'Link to Terms of Service',
+      env: 'PAYMENT_TERMS_OF_SERVCIE',
+      format: 'url',
+    },
+    cdnFqdn: {
+      default: 'accounts-static.cdn.mozilla.net',
+      doc: 'The domain name where the legal doc downloads are hosted.',
+      env: 'PAYMENT_LEGAL_DOWNLOAD_FQDN',
+      format: String,
+    },
+    httpResCacheLimit: {
+      default: 65536,
+      doc: 'The max number of entries in the redirect endpoint HTTP results cache.  0 means unlimited and the memory usage on the cache could reach the max of Map (1GB on V8)',
+      env: 'PAYMENT_LEGAL_DOWNLOAD_CACHE_LIMIT',
+      format: Number,
+    },
+  },
+  listen: {
+    host: {
+      default: '0.0.0.0',
+      doc: 'The ip address the server should bind',
+      env: 'IP_ADDRESS',
+      format: String,
+    },
+    port: {
+      default: 3031,
+      doc: 'The port the server should bind',
+      env: 'PORT',
+      format: 'port',
+    },
+    publicUrl: {
+      default: 'http://localhost:3031',
+      env: 'PUBLIC_URL',
+      format: 'url',
+    },
+    useHttps: {
+      default: false,
+      doc: 'set to true to serve directly over https',
+      env: 'USE_TLS',
+    },
+  },
+  logging: {
+    app: { default: 'fxa-payments-server' },
+    fmt: {
+      default: 'heka',
+      env: 'LOGGING_FORMAT',
+      format: ['heka', 'pretty'],
+    },
+    level: {
+      default: 'info',
+      env: 'LOG_LEVEL',
+    },
+    routes: {
+      enabled: {
+        default: true,
+        doc: 'Enable route logging. Set to false to trimming CI logs.',
+        env: 'ENABLE_ROUTE_LOGGING',
+      },
+      format: {
+        default: 'default_fxa',
+        format: ['default_fxa', 'dev_fxa', 'default', 'dev', 'short', 'tiny'],
+      },
+    },
+  },
+  oauth_client_id_map: {
+    default: {
+      dcdb5ae7add825d2: '123done',
+      '325b4083e32fe8e7': '321done',
+    },
+    doc: 'Mappings from client id to service name: { "id1": "name-1", "id2": "name-2" }',
+    env: 'OAUTH_CLIENT_IDS',
+    format: Object,
+  },
+  productRedirectURLs: {
+    default: {
+      '123doneProProduct': 'http://localhost:8080/',
+      fortressProProduct: 'http://localhost:9292/download',
+      prod_FUUNYnlDso7FeB: 'https://fortress-latest.dev.lcip.org/',
+      prod_GqM9ToKK62qjkK: 'https://123done-latest.dev.lcip.org/',
+      // todo get new prod_id for 123done stage
+      prod_FfiuDs9u11ESbD: 'https://123done-stage.dev.lcip.org',
+    },
+    doc: 'Mapping between product IDs and post-subscription redirect URLs',
+    env: 'PRODUCT_REDIRECT_URLS',
+    format: Object,
+  },
+  proxyStaticResourcesFrom: {
+    default: '',
+    doc: 'Instead of loading static resources from disk, get them by proxy from this URL (typically a special reloading dev server)',
+    env: 'PROXY_STATIC_RESOURCES_FROM',
+    format: String,
+  },
+  sentry: {
+    dsn: {
+      default: '',
+      doc: 'Sentry DSN',
+      env: 'SENTRY_DSN',
+      format: 'String',
```

Comment:
... unless this should be a `'url'` or something.
```suggestion
      format: String,
```